### PR TITLE
HTBHF-1915 Added validation of county field.

### PR DESCRIPTION
**Pull request by dwybourn**
Added validation for acceptance tests for county address field.

### CODE REVIEW

File: src/test/acceptance/features/manual-address.feature [hunk 1]
```diff
@@ -42,6 +42,10 @@ Feature: Address
     When I do not enter the 'town or city' of an address
     Then I am informed that I need to enter an address on the 'town or city' field
 
+  Scenario: Do not enter in the county
```
Comment (YetAnotherMatt) [inline src/test/acceptance/features/manual-address.feature]: I'd rather not make County a required field - we could default to a sensible value ('UK'? - do this in the claimant service, not here) if it's not provided.

Comment (dwybourn) [reply src/test/acceptance/features/manual-address.feature]: It's required for allpay, and will be printed on the letters. Maybe one for the UX team?

Comment (gethyn1) [reply src/test/acceptance/features/manual-address.feature]: What @YetAnotherMatt said!!

Comment (YetAnotherMatt) [reply src/test/acceptance/features/manual-address.feature]: I've spoken to the UX team, their preference is for it not to be there at all, and if it does have to be then for it to not be mandatory. If it's not present in the OS places response we won't be asking them to add it manually, so we'll have to handle an empty value anyway. Let's have the County field for now, not mandatory, and default to 'UK' in the claimant service if it's not provided. We can revisit whether the field is necessary once we've got OS places and postcodes.io integrated.

Comment (dwybourn) [reply src/test/acceptance/features/manual-address.feature]: Ok, will make optional

Comment (mikeSmith08) [reply src/test/acceptance/features/manual-address.feature]: Can we make sure that we have a test that makes sure an application is successful with no county then please?

File: src/test/common/steps/manual-address-steps.js [hunk 2]
```diff
@@ -61,6 +70,11 @@ Then(/^I am informed that I need to enter an address on the 'town or city' field
   await assertTownOrCityErrorFieldAndLink('Tell us your address')
 })
 
+Then(/^I am informed that I need to enter an address on the 'county' field$/, async function () {
```
Comment (mikeSmith08) [inline src/test/common/steps/manual-address-steps.js]: You can remove this method when it becomes optional.

File: src/test/acceptance/features/manual-address.feature (hunk 1 above)
Comment (dwybourn) [reply src/test/acceptance/features/manual-address.feature]: Field is now optional. Complete application test will come in next PR due to number of files it involves changing.

File: src/test/acceptance/features/manual-address.feature [hunk 3]
```diff
@@ -42,6 +46,7 @@ Feature: Address
     When I do not enter the 'town or city' of an address
     Then I am informed that I need to enter an address on the 'town or city' field
 
+
```
Comment (gethyn1) [inline src/test/acceptance/features/manual-address.feature]: Rogue new line

Comment (dwybourn) [reply src/test/acceptance/features/manual-address.feature]: Will remove in next PR.

**Pull request merged**